Sources/Sentry/SentryNetworkTracker.m

```diff
@@ -252,4 +245,20 @@ - (SentrySpanStatus)spanStatusForHttpResponseStatusCode:(NSInteger)statusCode
     return kSentrySpanStatusUndefined;
 }
 
+- (nullable NSDictionary *)addTraceHeader:(nullable NSDictionary *)headers
+{
+    id<SentrySpan> span = SentrySDK.currentHub.scope.span;
+    if (span == nil) {
+        return headers;
+    }
+
+    if (headers == nil) {
+        return @{ SENTRY_TRACE_HEADER : [span toTraceHeader].value };
```

Comment:
`[span toTraceHeader].value` guaranteed to return something that makes sense?
Or do we need to validate that before adding the header.

Reply:
It's guaranteed to return something meaningful as `toTraceHeader` is `nonnull` and also `.value`.